includes/data-stores/class-wc-admin-notes-data-store.php

```diff
@@ -250,6 +250,9 @@ private function save_actions( &$note ) {
 	public function get_notes( $args = array() ) {
 		global $wpdb;
 
+		$order   = isset( $args['order'] ) ? $args['order'] : 'DESC';
```

Comment:
For usage outside the context of a REST request, is there a need to sanitize/validate the args here?

Reply:
Good catch!  I don't think so.  I've also removed the `$page` and `$per_page` alterations that occurred here.

I can't find a place where they might have been used, but it looks like they were dealing with negative numbers, so probably passing `-1` to get all records.  Since the `apiFetch` method already handles this under the hood, I think we can safely remove, but I'll keep an eye on it and open a follow-up issue if this creates any problems.

Reply:
On second look, it does look like the data store is accessed directly for some functions like `WC_Admin_Notes::unsnooze_notes()`.  I've replaced this section with a default arguments pattern like we use in other data stores.